test/integration/licence_test.rb

```diff
@@ -69,7 +69,7 @@ class LicenceTest < ActionDispatch::IntegrationTest
         end
 
         within "#content" do
-          within ".page-header" do
```

Comment:
Always big ❤️ to removing test-only classes.

Reply:
>Unofficially approved 👍 I reckon we should hold off on this until Ian's point on the heading is sorted out. 

🙌🏼  Holding tight re the header visual.